### PR TITLE
#1256 Adding Lightstreams Network support

### PR DESCRIPTION
**Pull request by ggarri**
## Description

Support **Lightstreams Network**, Proof of Authority Ethereum compatible network for fast and secure file sharing and responsive DApps.

Homepage: https://lightstreams.network
Issue: #1256 

**Mainnet Network**
Block Explorer: https://explorer.lightstreams.io
RPC Endpoint: https://node.mainnet.lightstreams.io
ChainId: 163

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ### Release v5.0.15
 
 ### Bug
+
```
Comment (kvhnuke) [inline CHANGELOG.md]: can you please update the changelog?

File: src/wallets/hardware/ledger/appPaths.js [hunk 2]
```diff
@@ -158,6 +160,11 @@ const appList = [
     network: TT,
     prefixes: ["m/44'/1001'"],
     paths: [thundercore]
+  },
+  {
+    network: PHT,
+    prefixes: ["m/44'/60'"],
+    paths: [lightstreamsNetwork]
```
Comment (kvhnuke) [inline src/wallets/hardware/ledger/appPaths.js]: Also,  can you add the path to other wallets, such as `Trezor, keepkey..etc`

File: CHANGELOG.md (hunk 1 above)
Comment (ggarri) [reply CHANGELOG.md]: Done

File: src/wallets/hardware/ledger/appPaths.js (hunk 2 above)
Comment (ggarri) [reply src/wallets/hardware/ledger/appPaths.js]: Addressed at https://github.com/MyEtherWallet/MyEtherWallet/pull/1257/commits/dc31bedd0827b198d9dd64d12f6c9c44401ee541

**Comment by kvhnuke** [comment]
@ggarri thanks for the changes however, I dont think you have to add seperate paths for `keepkeyLightstream` and `ledgerLightStream` the reason why we have different paths for `Ethereum` for those is because when they initially released it, different hw wallets started using different paths but in your case you should just add `"m/44'/60'/0'/0"` `lightstreamsNetwork` path to everywhere

**Comment by ggarri** [comment]
Hi @kvhnuke I see where it was my confusion. As our network is fully ethereum compatible I was trying to use exact same paths for our network. I just pushed a new commit which updates the paths for Ledger and Keepkey to use "m/44'/60'/0'/0". 

Let me know if there is something else to do I will try to address it ASAP. Thanks :)

**Pull request merged**